web/src/test/manual/web/core/stress.html

```diff
@@ -51,7 +55,7 @@
 
 <body>
   <h2>KeymanWeb Sample Page - Core Testing</h2>
-  <p>This page is designed to stress-test the loading a KMX keyboard from blob.</p>
+  <p>This page is designed to stress-test the loading a KMX keyboard from blob. It will repeatedly load a .kmx using </p>
```

Comment:
```suggestion
  <p>This page is designed to stress-test the loading a KMX keyboard from blob. It will repeatedly load a .kmx using the KM_Core loader.</p>
```